services/cargohold/test/integration/Main.hs

```diff
@@ -24,6 +30,33 @@ data IntegrationConfig = IntegrationConfig
 
 instance FromJSON IntegrationConfig
 
+newtype ServiceConfigFile = ServiceConfigFile String
+    deriving (Eq, Ord, Typeable)
+
+instance IsOption ServiceConfigFile where
+    defaultValue = ServiceConfigFile "/etc/wire/cargohold/cargohold.yaml"
```

Comment:
for consistency with the other default paths, can we use `/etc/wire/cargohold/conf/cargohold.yaml` ?